openssl/src/ec.rs

```diff
@@ -631,6 +659,27 @@ impl EcPoint {
         }
         Ok(point)
     }
+
+    /// Creates point from a hexadecimal string representation
+    #[corresponds(EC_POINT_hex2point)]
+    #[cfg(not(boringssl))]
+    pub fn from_hex_str(
```

Comment:
I think we'd want this to take `&mut self` rather than being a static method taking the group.

Reply:
The definition is the same as EcPoint::from_bytes. For a &mut self method, it seems the name set_from_hex_str is more suited.